packages/ui-demo/src/Toast.stories.tsx

```diff
@@ -94,6 +94,17 @@ const Toasts = () => {
             );
           }}
         />
+        <Button
+          text="Dismissable"
+          onClick={() => {
+            const id = toast.show("Normal Toast", {
+              dismissable: true,
```

Comment:
Can we change this to `persistent` or something similar? Looking at this, i'd think dismissable as false would give me a permanently up toast (until the app dismisses it or something).

Reply:
Ah, great point. That could get confusing - fixed!